docs/streams/developer-guide/dsl-api.html

```diff
@@ -3134,6 +3192,49 @@ <h4><a id="streams_concepts_globalktable" href="#streams_concepts_globalktable">
                 </div>
             </div>
         </div>
+	
+	<div class="section" id="controlling-update-rate">
+            <span id="streams-developer-guide-dsl-suppression"></span><h2><a class="toc-backref" href="#id32">Controlling KTable update rate</a><a class="headerlink" href="#controlling-update-rate" title="Permalink to this headline"></a></h2>
+	    <p>A KTable is logically a continuously updated table.
+	       These updates make their way to downstream operators whenever new data is available, ensuring that the whole computation is as fresh as possible.
+	       Logically speaking, most programs describe a series of transformations, and the update rate is not a factor in the program behavior.
+	       In these cases, the rate of update is more of a performance concern.
+	       Operators are able to optimize both the network traffic (to the Kafka brokers) and the disk traffic (to the local state stores) by adjusting
+	       the commit interval and batch size configurations.
```

Comment:
nit: commit interval and cache size

Reply:
I actually meant batch size. As in, they can manipulate the batching configs on the producer, and on rocks. Cache size is one way to influence this behavior, but there are several others.

I considered enumerating them here, but this isn't a performance tuning doc. I just wanted to provide a little context.